docs/extensions/METADATA.md

```diff
@@ -80,7 +88,30 @@ For more advanced use cases, the extension can also:
 },
 ```
 
-The vm metadata section should define either `image` or `composefile`. When you use `image`, a default compose file is generated for the extension.
+The composefile would then look like (for exampe with a volume definition):
+
+```yaml
+services:
+  myExtension:
+    image: ${DESKTOP_PLUGIN_IMAGE}
+    volumes:
+      - /host/path:/container/path
+```
+
+### Using the docker socket from your extension backend
+
+Docker extensions can invoke Docker commands directly from the frontend with the SDK. In some cases, it is useful to also interact with the Docker engine from the backend. Extension backend containers can mount the Docker socket and use it to interact with the Docker engine from the extension backend logic. (Learn more about the [Docker engine socket](https://docs.docker.com/engine/reference/commandline/dockerd/#examples))
+
+However, when mounting the docker socket from an extension container that lives in the Desktop virtual machine, you want to mount the Docker socket from inside the VM, and not mount `/var/run/docker.sock` from the host filsystem (using the Docker socket from the host can lead to permission issues in containers).
+In order to do so, you can use `/var/run/docker.sock.raw` and Docker Desktop will treat this specifically to mount the socket that lives in the Desktop VM, and not from the host.
```

Comment:
Not sure about this sentence. I've added a suggestion, but not sure if this then changes the meaning of the sentence?
```suggestion
In order to do so, you can use `/var/run/docker.sock.raw`. Docker Desktop mounts the socket that lives in the Desktop VM, and not from the host.
```